app-laptop/i8kutils/i8kutils-1.42.ebuild

```diff
@@ -1,22 +1,24 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=4
+EAPI=5
 
 inherit systemd toolchain-funcs
 
 DESCRIPTION="Dell Inspiron and Latitude utilities"
-HOMEPAGE="http://packages.debian.org/sid/i8kutils"
-SRC_URI="mirror://debian/pool/main/i/${PN}/${P/-/_}.tar.gz"
+HOMEPAGE="https://launchpad.net/i8kutils"
+SRC_URI="https://launchpad.net/i8kutils/trunk/${PV}/+download/${P/-/_}.tar.xz"
+
+S="${WORKDIR}/${PN}"
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="-* amd64 x86"
```

Comment:
Don't add new versions with stable keywords. You can use `ekeyword ~all i8kutils-1.42.ebuild` to automatically reduce it to `~amd64 ~x86`.